enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/AuthFilter.java

```diff
@@ -54,6 +56,8 @@ public class AuthFilter implements Filter {
     private List<Authenticator> authenticators = new ArrayList<>();
     private static final Logger log = LogManager.getLogger(AuthFilter.class);
 
+    private static ConcurrentHashMap<String, JWKSet> jwksMap = new ConcurrentHashMap<>();
```

Comment:
Any reason, jwksMap is initialized in AuthFilter and pass it into `JWTAuthenticator` -> `JWTValidator` without directly initialising it in JWTValidator? Is that for any future reference?

Reply:
no future reference in mind, its just how I thought of it. What you suggested would be better

Reply:
LGTM now, after moving the map to JWTValidator.